source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -0,0 +1,62 @@
+# NVDAObjects/UIA/winConsoleUIA.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2019 Bill Dengler
+
+import time
+import textInfos
+import UIAHandler
+
+from scriptHandler import script
+from winVersion import winVersion
+from . import UIATextInfo
+from ..behaviors import Terminal
+
+
+class consoleUIATextInfo(UIATextInfo):
+	_expandCollapseBeforeReview = False
+
+	def __init__(self, obj, position, _rangeObj=None):
+		super(consoleUIATextInfo, self).__init__(obj, position, _rangeObj)
+		if position == textInfos.POSITION_CARET:
+			if winVersion.build >= 18362:  # Windows 10 version 1903
+				# The UIA implementation in 1903 causes the caret to be
+				# off-by-one, so move it one position to the right
+				# to compensate.
+				self._rangeObj.MoveEndpointByUnit(
+					UIAHandler.TextPatternRangeEndpoint_Start,
+					UIAHandler.NVDAUnitsToUIAUnits['character'],
```

Comment:
Can you please use textInfos.UNIT_CHARACTER rather than the literal string here.